### ISSUE TITLE
autobutcher ticks error?

### ISSUE THREAD

**Issue by MechanosG** [DFHack/dfhack#3541]
Attempting to change the ticks, spits an error about requiring a positive integer, then throws the autobutcher help info into console.

Steps to reproduce:
```
enable autobutcher
autobutcher ticks 1000
```
Result is this in the console:
```
lua: number of ticks must be a positive integer: 1000
```
Followed by the help description stuff.  Opening gui/autobutcher confirms that the tick isn't getting changed.  But it seems the GUI is able to set it to 1000 without issue.

**Comment by myk002** [comment]
I was actually considering taking the ticks configurability out. Out of curiosity, what did you need to change it for?

**Comment by MechanosG** [comment]
Honestly, I could live without it.  I initially set it to 1000 because I wasn't sure what the default was, and wasn't sure how long 1000 ticks was.  Then later, learned the default is 6000 ticks (I believe?).  Which is only like 5 in-game days, or 3-4 minutes irl in a bogged down fort (or lower end PC like mine), or 1-2 minutes normally.  I probably don't need it any faster than that lol.  I've been using it with the error off and on since like..  February, so I've already been using it at 6000 ticks for a while.  I just got tired of the console spam while debugging something else today, and figured I should make a bug report since I commented out setting the ticks in my script, and won't be seeing it anymore.
